understanding/20/contrast-enhanced.html

```diff
@@ -96,15 +96,32 @@ <h2>Intent of Contrast (Enhanced)</h2>
          in order to get a particular look. 
       </p>
       
-      <p>Although this Success Criterion only applies to text, similar issues occur for content presented in charts, graphs, diagrams, and other non-text-based information which is covered by 
-         <a href="../21/non-text-contrast.html">Success Criterion 1.4.11 Non-Text Contrast</a>.         
+      <div class="note">
+         
+         <p>Images of text do not scale as well as text because they tend to pixelate. It is also
+            harder to change foreground and background contrast and color combinations for images
+            of text, which is necessary for some users. Therefore, we suggest using text wherever
+            possible, and when not, consider supplying an image of higher resolution.
```

Comment:
```suggestion
            possible (see <a href="images-of-text">1.4.5: Images of Text</a>, and when not, consider supplying an image of higher resolution.
```

Reply:
ed: missing closing paren in the suggestion